# sets the active degrees of freedom of the robot (used by planners to get configuration space)

int32 robotid
ActiveDOFs active
---
